soc/xtensa/intel_adsp/cavs_v18/include/soc/platform.h

```diff
@@ -0,0 +1,25 @@
+/* SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright(c) 2016 Intel Corporation. All rights reserved.
+ *
+ * Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
+ *         Keyon Jie <yang.jie@linux.intel.com>
+ *         Xiuli Pan <xiuli.pan@linux.intel.com>
+ */
+
+#ifndef __PLATFORM_PLATFORM_H__
+#define __PLATFORM_PLATFORM_H__
+
+#define PLATFORM_MASTER_CORE_ID			0
```

Comment:
can we use 'primary' instead of 'master' in this PR? This was changed in the SOF code already, this seems to be an older version.